StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -354,6 +355,24 @@ extension LinkAccountPickerViewController: LinkAccountPickerBodyViewDelegate {
 
         let selectedPartnerAccount = selectedAccountTuple.partnerAccount
 
+        if let drawerOnSelection = selectedAccountTuple.accountPickerAccount.drawerOnSelection {
+            let genericInfoViewController = GenericInfoViewController(
+                genericInfoScreen: drawerOnSelection,
+                panePresentationStyle: .sheet,
+                didSelectPrimaryButton: { [weak self] in
+                    self?.lastDrawerOnSelectionViewController?.dismiss(animated: true)
+                },
+                didSelectURL: { url in } // TODO(kgaidis): fix handling URL
```

Comment:
this is intentional...will fix later because don't want to blow up the PR size (this is merged into a feature branch)